### PR TITLE
featc(unbind): Impl unbinding as an finalizer action of app CR (AEROGEAR-9088)

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
https://issues.jboss.org/browse/AEROGEAR-9088

## What
* Remove unbind type
* Add impl to unbind app into finalizer of app type
* update the readme

## Verification Steps
 
1. Use this PR and the image `docker.io/cmacedo/mobile-security-service-operator:AEROGEAR-9088`
2. Create all with `make create-all`
3. Check that all is running with success then create a new namespace `on new-project myapptest` 
4. Create app by `make create-app`
5. Check that the app was created in the REST Service
6. Run `make delete-app` and check that the app from the REST and configMap were removed with success when the CR was deleted.
 
## Checklist:

- [X] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Finished task
- [ ] TODO

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -133,14 +131,14 @@ create-db-only:
 	@echo Creating Mobile Security Service Database only:
 	kubectl apply -f deploy/crds/mobile-security-service_v1alpha1_mobilesecurityservicedb_cr.yaml
 
-.PHONY: delete-app
-delete-app:
+.PHONY: delete-service-app-and-db
+delete-service-app-and-db:
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply Makefile]: done

[the remark this answers is not in the document]
Comment (davidffrench) [reply Makefile]: This line(134 & 135) should be changed to `delete-service-and-db:` to be consistent. There are still 4 references to unbind in the make file, as said in 2 other comments. These need to be removed.

**Comment by camilamacedo86** [comment]
Hi @davidffrench, 

Sorry, I forgot to remove the unbind ones and I renamed the make commands as requested. I hope that all is fine now. Please, let me know if we can move forward with.

**Comment by camilamacedo86** [comment]
Hi @davidffrench,

Really thank you for your time and review.

**Pull request merged**